src/openrct2/network/Socket.h

```diff
@@ -33,6 +33,13 @@ enum class NetworkReadPacket : int32_t
     Disconnected
 };
 
+typedef struct SRVRecord {
+    u_int16_t priority;
+    u_int16_t weight;
+    u_int16_t port;
+    char dname[255];
+} SRVRecord;
```

Comment:
This seems C-like, we can probably use something a little more C++-zy.
We also use `uint16_y` instead of `u_int16_t`
```suggestion
struct SRVRecord {
    uint16_t priority;
    uint16_t weight;
    uint16_t port;
    char dname[255];
};
```